package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jupyter",
     "displayName": "Jupyter",
-    "version": "2022.2.100",
+    "version": "2022.2.101",
```

Comment:
This will ensure the new nightly build will be a newer version.
The format is `2022.02.101MMDDHHm`